lldap_graphql/__init__.py

```
@@ -0,0 +1,204 @@
+        self.graphql_url: str = self.base_url + HTTP_GRAPHQL_ENDPOINT
+        self.auth_url: str = self.base_url + HTTP_AUTH_ENDPOINT
+        self.refresh_url: str = self.base_url + HTTP_REFRESH_ENDPOINT
+        self.username: str = username
+        self.password: str = password
+        self.token: str | None = None
+        self.refresh_token: str | None = None
+
+    def get_token(self) -> None:
+        """Get a token from the lldap GraphQL API."""
+        payload: dict[str, str] = {"username": self.username, "password": self.password}
+        response: requests.Response = requests.post(self.auth_url, json=payload)
```

Pylint (3.3.7)

[warning] Missing timeout argument for method 'requests.post' can cause your program to hang indefinitely

(W3101)